WalletWasabi/WabiSabi/Backend/Rounds/CoinJoinStorage/InMemoryCoinJoinIdStore.cs

```diff
@@ -40,4 +41,27 @@ public static InMemoryCoinJoinIdStore LoadFromFile(string filePath)
 		var store = new InMemoryCoinJoinIdStore(lines);
 		return store;
 	}
+
+	public void ImportWW1CoinJoinsToWW2(string ww1CoinJoinsFilePath, string coinJoinIdStoreFilePath)
```

Comment:
Can be static?

Reply:
Move this to CoinJoinIdStore.

Reply:
With refactored logic, now it can be, yes.